RACK-Ontology/ontology/DO-330/OBJECTIVE_2.csv

```diff
@@ -0,0 +1,77 @@
+description,entityURL,generatedAtTime,identifier,invalidatedAtTime,title,definedIn_identifier,satisfiedBy_identifier,dataInsertedBy_identifier,wasAttributedTo_identifier,wasDerivedFrom_identifier,wasGeneratedBy_identifier,wasImpactedBy_identifier,wasRevisionOf_identifier
+"Tool Executable Object Code is produced. ","","","T-2-7","","","","","","","","","",""
```

Comment:
Not sure why, but this CSV files uses a lot of empty strings for the missing fields, whereas the other ones just put nothing.

This may cause problems?

Reply:
Ah. Good find. Didn't cause any problems, but I cleaned up the empty strings anyway.